test/github-issues/issue-215.test.js

```diff
@@ -0,0 +1,81 @@
+'use strict'
+
+const FakeTimers = require('@sinonjs/fake-timers')
+const t = require('tap')
+const test = t.test
+const Fastify = require('fastify')
+const rateLimit = require('../../index')
+
+t.beforeEach(t => {
+  t.context.clock = FakeTimers.install()
+})
+
+t.afterEach(t => {
+  t.context.clock.uninstall()
+})
+
+test('issue #215 - when using local store, 2nd user is not being rate limited when time window is passed for the 1st user', async t => {
```

Comment:
Not 100% on the test description, a tad long?

Reply:
Actually it should be:
```suggestion
test('issue #215 - when using local store, 2nd user should not be rate limited when time window is passed for the 1st user', async t => {
```

but it's still long, do you have a suggestion?

Reply:
That's clearer!